CenterForOpenScience/osf.io
# [Feature] Rework prereg page [OSF-7291]

## Purpose
Prereg page needed streamlining. 


## Changes
1. Change main text per specs
2. Make page work with not logged in users, prompt to create account if click `Preregister` button


BEFORE:
<img width="1438" alt="screen shot 2017-07-21 at 7 07 33 pm" src="https://user-images.githubusercontent.com/1322421/28485478-e7195b5a-6e47-11e7-84b5-5b16e3d6b56e.png">
AFTER:
<img width="1440" alt="screen shot 2017-07-21 at 6 51 24 pm" src="https://user-images.githubusercontent.com/1322421/28485482-edc0a4a4-6e47-11e7-9e71-ebb5c024689e.png">
AFTER Not logged in:
 
<img width="1440" alt="screen shot 2017-07-21 at 6 51 39 pm" src="https://user-images.githubusercontent.com/1322421/28485489-fc298498-6e47-11e7-9555-8ba3952d899b.png">

BEFORE Mobile:
<img width="401" alt="screen shot 2017-07-21 at 7 05 31 pm" src="https://user-images.githubusercontent.com/1322421/28485460-c06910a4-6e47-11e7-8d91-2b673be52561.png">
AFTER Mobile: 
<img width="401" alt="screen shot 2017-07-21 at 7 05 52 pm" src="https://user-images.githubusercontent.com/1322421/28485463-c5f9d47c-6e47-11e7-90a4-8722fba863b6.png">

 

## Side effects
1. Give buttons max width, single button on page took up width of the page before change
2. Make button language between small and large views the same
3. Make mobile buttons scale better and not cut off.
4. Fix button action drop downs to scale better with buttons (note they are not perfectly aligned in mobile, but are better than they were previously
5. ERPC mobile view still showed buttons to make a registration, which lead to a broken page. I removed those buttons. 
6. Furthermore, I split Prereg and ERPC from using the same template, since there was way too much jerryrigging occurring to make it work.




## Ticket
https://openscience.atlassian.net/browse/OSF-8291

## QA Notes
1. Make sure ERPC and Prereg pages work properly
2. Check all side effects to ensure they look right
~3. Make sure scaling works better than it did before, still not perfect for some things~
3. Scaling should work 100% now. All buttons and drops from buttons should align.
4. Ensure all buttons still work
5. Ensure clicking `Preregister` when not logged in redirects back to prereg (both for logging in and making a new account)

## Tests
I fixed some failing tests, removed one that was no longer relevant, and added one for not-logged-in users
